src/components/App/App.js

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
+import { Provider } from 'react-redux';
 import ContentHolderComponent from '../ContentHolder/ContentHolderComponent';
 import { store } from '../../store/store';
-import { Provider } from 'react-redux';
+import { BrowserRouter as Router, Switch, Route, Redirect } from 'react-router-dom';
+import PageNotFound from '../PageNotFound/PageNotFound';
 
-const App =  () => {
+const App = () => {
   return (
     <React.StrictMode>
       <div className="container">
-      <Provider store={store}>
-          <ContentHolderComponent/>
+        <Provider store={store}>
+          <Router>
+          <Switch>
+            <Route exact path="/home" ><ContentHolderComponent/></Route>
```

Comment:
let's change it like this: 

`<Route exact path="/home" component={ContentHolderComponent} />`

and next question, why did you add ONE component for different routes?? pls fix it!)